react/StarIcon/StarIcon.less

```diff
@@ -9,3 +12,36 @@
   50% { transform: scale(1.1); }
   100% { transform: scale(1); }
 }
+
+.svg {
+  position: relative;
+  top: 0.05em;
+}
+
+.heroSvg {
```

Comment:
It feels like we need an interface to set the `icon-scale` from the component rather than having to duplicate/override all these mixins. Is that something you looked into already?

Reply:
We were trying to resize the stars without changing the size of other icons and make it uncoupled.

Reply:
For me, I think the main thing was to make it really explicit so it's easy for people to make tweaks to individual sizes, if necessary.

Reply:
I'm open to the mixin idea, I'm just a little concerned that it hides too much away and makes it hard to adjust the output.